openstack/nova-hypervisor-agents/templates/hypervisors-kvm-daemonset.yaml

```diff
@@ -0,0 +1,296 @@
+    spec:
+      terminationGracePeriodSeconds: {{ $hypervisor.default.graceful_shutdown_timeout | default .Values.defaults.default.graceful_shutdown_timeout | add 5 }}
+      hostNetwork: true
+      hostPID: true
+      hostIPC: true
+      securityContext:
+        supplementalGroups: [476, 477]
+        fsGroup: 42436
+      tolerations:
+      {{- range $k, $v :=  merge $hypervisor.pod.tolerations .Values.pod.tolerations }}
+      - # {{ $k }}
+      {{- toYaml $v | nindent 8 }}
+      {{- end }}
+      nodeSelector:
+      {{- range $k, $v := $hypervisor.pod.nodeSelector }}
+        {{ $k }}: {{ $v }}
+      {{- end }}
+      hostAliases:
+      {{- range $ip := .Values.rabbitmq.externalIPs }}
+      - ip: {{ $ip }}
+        hostnames:
+        - {{ default "rabbitmq" $.Values.rabbitmq.nameOverride | printf "nova-%s" | trunc 63 | replace "_" "-" | trimSuffix "-" }}
+      {{- end }}
+      initContainers:
+{{- if not .Values.hypervisor_on_host }}
+      - name: load-kernel-mod
+        image: {{ required ".Values.global.dockerHubMirror is missing" .Values.global.dockerHubMirror }}/library/busybox
+        command:
+        - modprobe
+        - kvm
+        volumeMounts:
+        - mountPath: /lib/modules
+          name: modules
+          readOnly: true
+      - name: fix-permission-instance-volume
+        image: {{ required ".Values.global.dockerHubMirror is missing" .Values.global.dockerHubMirror }}/library/busybox
+        command: ["sh", "-c", "chown -R 42436:42436 /var/lib/nova/instances"]
+        volumeMounts:
+        - mountPath: /var/lib/nova/instances
+          name: instances
+{{- end }}
+      containers:
+        - name: nova-compute
+          image: {{ required ".Values.global.registry is missing" .Values.global.registry }}/{{ .Values.imageName }}:{{ .Values.imageVersion | required "Please set imageVersion similar" }}
+          imagePullPolicy: IfNotPresent
+          securityContext:
+            privileged: true
+            runAsUser: 42436
+            supplementalGroups: [477]
+          command:
+          - "nova-compute"
+          env:
+          {{- if .Values.sentry.enabled }}
+          - name: SENTRY_DSN
+            valueFrom:
+              secretKeyRef:
+                name: sentry
+                key: nova.DSN.python
+            {{- end }}
+{{- if or $hypervisor.python_warnings .Values.python_warnings }}
+          - name: PYTHONWARNINGS
+            value: {{ or $hypervisor.python_warnings .Values.python_warnings | quote }}
+{{- end }}
+          volumeMounts:
+          - mountPath: /var/lib/nova/instances
+            name: instances
+          - mountPath: /var/lib/libvirt
+            name: libvirt
+            readOnly: true
+          - mountPath: /run
+            name: run
+          - mountPath: /etc/nova
+            name: nova-etc
+          - mountPath: /etc/sudoers.d/nova
+            name: nova-etc
+            subPath: sudoers-nova
+            readOnly: true
```

Comment:
`nova-etc` is now a projected volume and does not include the subPath `sudoers-nova` anymore. Not sure how to handle this - I guess we also have to add the nova-etc ConfigMap with a different name?